tests/webtests/EditServiceTest.php

```diff
@@ -74,8 +75,7 @@ public function test_privacy_questions_admin_toggle()
             ->form();
         $form->setValues($formData);
         self::$pantherClient->executeScript("document.getElementsByClassName('service-form').item(0).submit();");
-        self::$pantherClient->wait(3);
-        self::assertOnPage('Your changes were saved!');
+        self::$pantherClient->wait(10)->until( WebDriverExpectedCondition::elementTextContains(WebDriverBy::cssSelector('body'), 'Your changes were saved!'));
```

Comment:
Do you need the `wait` in conjunction with `until`? Wouldn't `until` wait as long as need be?

Reply:
https://github.com/php-webdriver/php-webdriver/wiki/HowTo-Wait

```
// Wait for at most 10s and retry every 500ms if it the title is not correct.
$driver->wait(10, 500)->until(
  WebDriverExpectedCondition::titleIs('My Page')
);
```

Reply:
TIL!